packages/cardano-services/package.json

```
@@ -52,7 +52,7 @@
     "run:http-server:debug": "npx nodemon --legacy-watch --exec 'node -r ts-node/register --inspect=0.0.0.0:9229 src/run.ts'",
     "run:tx-worker": "ts-node --transpile-only src/startWorker.ts",
     "run:tx-worker:debug": "npx nodemon --legacy-watch --exec 'node -r ts-node/register --inspect=0.0.0.0:9229 src/startWorker.ts'",
-    "test": "jest --runInBand -c ./jest.config.js",
+    "test": "jest --runInBand -c ./jest.config.js --updateSnapshot",
```

This will not fail when something breaks. You can just run `yarn test --updateSnapshot` when you want to update the snapshots

```suggestion
    "test": "jest --runInBand -c ./jest.config.js",
```